### ISSUE TITLE
The memmap crate is no longer maintained.

### ISSUE THREAD

**Issue by vext01** [CensoredUsername/dynasm-rs#52]
I know you are aware of this @CensoredUsername, however `cargo audit` has just recently started complaining about it:
```
Crate:         memmap
Version:       0.7.0
Warning:       unmaintained
Title:         memmap is unmaintained
Date:          2020-12-02
ID:            RUSTSEC-2020-0077
URL:           https://rustsec.org/advisories/RUSTSEC-2020-0077
Dependency tree: 
memmap 0.7.0
├── ykview 0.1.0
├── yktrace 0.1.0
│   ├── ykview 0.1.0
│   ├── ykrt 0.1.0
│   └── ykcompile 0.1.0
│       └── ykrt 0.1.0
└── dynasmrt 1.0.0
    └── ykcompile 0.1.0
```

I think you said you had another library in mind. Is it time to switch to it perhaps?

**Comment by CensoredUsername** [comment]
Linking the relevant issue here for posterity: danburkert/memmap-rs/issues/90

**Comment by CensoredUsername** [comment]
I don't have the time or resources to maintain that crate myself. In the thread there seems to be a consensus to wait a week on if this advisory triggers any action to try to move things. As the crate is relatively small and just does one thing properly there's not really a risk of security issues with it. It's just annoying right now as it blocks fixing a bug (make MAP_ANON maps non MAP_SHARED).

**Comment by CensoredUsername** [comment]
Current candidate for replacement is `memmap2`. I also already merged a fix for the MAP_SHARED | MAP_ANON issue there.

**Comment by vext01** [comment]
Sounds good! Shall we go ahead with the switch?

**Comment by vext01** [comment]
Any thoughts on this? Our weekly audit is still flagging this as a problem.

**Comment by CensoredUsername** [comment]
Whoops I completely forgot about this, life got a bit in the way. I need to do a bit of branching to get that in right now because there's some stuff on dev staged that needs a new major release but also some more evaluation first.

**Comment by CensoredUsername** [comment]
Fixed in 1.0.1, which was just released.

**Comment by vext01** [comment]
Great! Thanks!
